crates/bitwarden-auth/src/registration.rs

```diff
@@ -33,4 +49,99 @@ impl RegistrationClient {
         let api_client = &client.get_api_configurations().await.api_client;
         // Do API request here. It will be authenticated using the client's tokens.
     }
+
+    /// Initializes a new cryptographic state for a user and posts it to the server; enrolls the
+    /// user to key connector unlock.
+    pub async fn post_keys_for_key_connector_registration(
```

Comment:
Relaying feedback I got on my PR here, we probably want unit tests on this. Vault has some examples for this, but it generally involves moving the impl into a separate function that you pass in the "api client" dependency and other parameters to, then the test can replace that with a mocked api client.